search-app/src/cmr/search/services/parameters/parameter_validation.clj

```diff
@@ -110,9 +110,10 @@
    :temporal (conj exclude-plus-and-or-option :limit-to-granules)
    :revision-date cpv/and-option
    :created-at cpv/and-option
+   :granule-created-at cpv/and-option
```

Comment:
Oh right - this request isn't flowing through the usual way. The request is working as expected, but it isn't using this validation